### PR TITLE
Fixup make targets

### PR DESCRIPTION
**Pull request by Rob-Johnson**
This allows us to build PaaSTA with our internal Jenkins until we remove the remaining internal dependencies.

I've renamed the make targets to try and make it more obvious these are 'yelpy' things.

The final piece is to tell Jenkins to run ``make itest_%_yelp``

### CODE REVIEW

**Comment by Rob-Johnson** [comment]
I've broken other things here. WIP.

File: Makefile [hunk 1]
```diff
@@ -26,7 +26,7 @@ itest: test
 	tox -e paasta_itests
 
 # See the makefile in yelp_package/Makefile for packaging stuff
-itest_%:
+itest_%_yelp:
```
Comment (mrtyler) [inline Makefile]: Does this work? Maybe yelp_itest_% or itest_yelp_% will work?

File: debian/rules [hunk 2]
```diff
@@ -19,7 +19,7 @@ override_dh_auto_build:
 	true
 
 override_dh_installdocs:
-	tox -e manpages
+	tox -i http://pypi.yelpcorp.com -e manpages
```
Comment (mrtyler) [inline debian/rules]: This seems like a step backward. If anything I'd expect the internal pypi to be hard-coded at a higher level so it's used everywhere and not just for this particular manpages invocation.

File: requirements.txt [hunk 3]
```diff
@@ -20,7 +20,7 @@ lockfile==0.9.1
 -e git://github.com/yelp/marathon-python.git@e7e2116159528c45afdbb8b01c03fbd0333e0108#egg=marathon-python
 mccabe==0.3.1
 mesos.cli==0.1.3
-mesos.interface==0.22.1.2
```
Comment (mrtyler) [inline requirements.txt]: We don't need whatever was in that .2 patch release?

File: yelp_package/Makefile [hunk 4]
```diff
@@ -26,7 +26,9 @@ build_lucid_docker:
 	[ -d ../dist ] || mkdir ../dist
 	cd dockerfiles/lucid/ && docker build -t "paastatools_lucid_container" .
 package_lucid_yelp: build_lucid_docker
-	cp setup.py ../; cp requirements.txt ../
+	cp ../requirements.txt requirements.txt.bak
+	cat requirements-extras.txt >> ../requirements.txt
+	mv requirements.txt.bak ../requirements.txt
```
Comment (mrtyler) [inline yelp_package/Makefile]: I'm pretty sure the result of this is to clobber the changes you just made to ../requirements.txt. I'm pretty lost about what is happening. Also we don't need setup.py in .. anymore?

File: yelp_package/Makefile [hunk 5]
```diff
@@ -37,7 +36,6 @@ build_trusty_docker:
 	[ -d ../dist ] || mkdir ../dist
 	cd dockerfiles/trusty/ && docker build -t "paastatools_trusty_container" .
 package_trusty_yelp: build_trusty_docker
-	cp setup.py ../; cp requirements.txt ../
```
Comment (Rob-Johnson) [inline yelp_package/Makefile]: this can go now that I've moved the internal deps to tox.ini

File: Makefile (hunk 1 above)
Comment (Rob-Johnson) [inline Makefile]: yes, this seems to work. I can run ``make itest_trusty_yelp`` and ``make itest_lucid_yelp`` and it does the right thing

**Pull request merged**